### ISSUE TITLE
1.6-rc.4 - client error w/import. 1.6-rc.3 works.

### ISSUE THREAD

**Issue by benlavalley** [meteor/meteor#9197]
Updated from 1.5.2.2 to 1.6-rc.5. OS X 10.12.6. Everything appeared to work OK until I went to load my app and got this error in the browser:

```
10:41:19.089 modules-runtime.js?hash=8587d188e038b75ecd27ed2469a52b269e38fb62:231 Uncaught Error: Cannot find module '/imports/lib/collections/restaurantMenuDishes.js'
    at makeMissingError (modules-runtime.js?hash=8587d188e038b75ecd27ed2469a52b269e38fb62:231)
    at fileEvaluate (modules-runtime.js?hash=8587d188e038b75ecd27ed2469a52b269e38fb62:328)
    at require (modules-runtime.js?hash=8587d188e038b75ecd27ed2469a52b269e38fb62:238)
    at globalHelpers.js (globalHelpers.js:1)
    at fileEvaluate (modules-runtime.js?hash=8587d188e038b75ecd27ed2469a52b269e38fb62:343)
    at require (modules-runtime.js?hash=8587d188e038b75ecd27ed2469a52b269e38fb62:238)
    at 01_commonImports.js (01_commonImports.js:1)
    at fileEvaluate (modules-runtime.js?hash=8587d188e038b75ecd27ed2469a52b269e38fb62:343)
    at require (modules-runtime.js?hash=8587d188e038b75ecd27ed2469a52b269e38fb62:238)
    at app.js?hash=21e1c596eedf7d6d4861fb6fc1c1c1df94152805:40339

```
I downgraded back to 1.6-rc.2 and found things were working. I then upgrade to 1.6-rc.3 -- still worked. 

Upgrading to 1.6-rc.4 yielded the failed import again. 

Currently downgraded to 1.6-rc.3. 

Changed import load order and it still seems to error on other collections I'm trying to import from that folder. Haven't tried to troubleshoot further if other types of modules will be affected. It appears a number of other modules are importing OK before the error is thrown.

Spent a good bit of time trying to figure this out, forgive me for not sharing a sample project w/reproduction for now - if this proves to be difficult to track down I'll try to spend time on that later today.

Some hopefully useful tidbits:

```
billgatesmacbookpro:web ben$ meteor update --release=1.6-rc.4
                                                                                   
Changes to your project's package version selections from updating the release:
                                              
accounts-facebook     upgraded from 1.3.0-rc.3 to 1.3.0-rc.4
accounts-password     upgraded from 1.5.0-rc.3 to 1.5.0-rc.4
accounts-twitter      upgraded from 1.4.0-rc.3 to 1.4.0-rc.4
dynamic-import        upgraded from 0.2.0-rc.3 to 0.2.0-rc.4
facebook-oauth        upgraded from 1.4.0-rc.3 to 1.4.0-rc.4
http                  upgraded from 1.3.0-rc.3 to 1.3.0-rc.4
less                  upgraded from 2.7.11-rc.3 to 2.7.11-rc.4
meteor-base           upgraded from 1.2.0-rc.3 to 1.2.0-rc.4
minifier-js           upgraded from 2.2.0-rc.3 to 2.2.0-rc.4
oauth                 upgraded from 1.2.0-rc.3 to 1.2.0-rc.4
oauth1                upgraded from 1.2.0-rc.3 to 1.2.0-rc.4
oauth2                upgraded from 1.2.0-rc.3 to 1.2.0-rc.4
reactive-dict         upgraded from 1.2.0-rc.3 to 1.2.0-rc.4
shell-server          upgraded from 0.3.0-rc.3 to 0.3.0-rc.4
standard-minifier-js  upgraded from 2.2.0-rc.3 to 2.2.0-rc.4
```

My guess is the issue lies in the recent changes to dynamic-import 0.2.0-rc.4. 

Figured I'd get this posted sooner rather than later as I am thinking this issue is likely to affect other apps. 

Pretty thrilled my upgrade to 1.6 worked in general and I'd imagine this will be an easy fix :)

Package.json:
[package.json.txt](https://github.com/meteor/meteor/files/1368970/package.json.txt)

.meteor/versions from working 1.6-rc.3:
[versions-rc3.txt](https://github.com/meteor/meteor/files/1369048/versions-rc3.txt)

.meteor/versions.txt from NOT working 1.6-rc.4:
[versions-rc4.txt](https://github.com/meteor/meteor/files/1369051/versions-rc4.txt)

**Comment by benjamn** [comment]
What about `1.6-rc.5`?

**Comment by benlavalley** [comment]
> Updated from 1.5.2.2 to **1.6-rc.5**. OS X 10.12.6. Everything appeared to work OK until I went to load my app and got this error in the browser... 

RC5 was what I went to at first and had the error, then I started downgrading to try and help narrow down where the issue was introduced, and landed on RC4.

**Comment by stolinski** [comment]
I'm getting this same error with 1.6-rc.5

**Comment by stolinski** [comment]
Fixed for me in 1.6-rc.6

**Comment by skirunman** [comment]
` 1.6-rc.6` fixed the import issue we were seeing in ` 1.6-rc.4` and ` 1.6-rc.5`, thanks!

**Comment by benlavalley** [comment]
Knew it would be a quick fix - resolved for me in 1.6-rc.6. Good boy points earned, @benjamn!

**Comment by benjamn** [comment]
I'm 32, but I'll pass those points along to [my dog](https://www.instagram.com/inspector_beckett/).
